streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBWindowStore.java

```diff
@@ -72,8 +73,12 @@ public void put(final K key, final V value) {
     @Override
     public void put(final K key, final V value, final long timestamp) {
         maybeUpdateSeqnumForDups();
-
-        bytesStore.put(WindowKeySchema.toStoreKeyBinary(key, timestamp, seqnum, serdes), serdes.rawValue(value));
+        try {
+            bytesStore.put(WindowKeySchema.toStoreKeyBinary(key, timestamp, seqnum, serdes), serdes.rawValue(value));
+        } catch (final ProcessorStateException e) {
```

Comment:
As above. Not sure if we need this, as the store should be wrapped with `MeteredWindowStore`.

Reply:
Done